### PR TITLE
update ambassador page

### PR DESCRIPTION
**Pull request by mribeirodantas**
- **Update ambassador page**
- **Add alumni section in our ambassadors page**

### CODE REVIEW

File: src/pages/our_ambassadors.astro [hunk 1]
```diff
@@ -547,4 +547,7 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
       </AmbassadorCard>
     </div>
   </div>
+
+  <h1>Alumni</h1>
+  Soon
```
Comment (christopher-hakkaart) [inline src/pages/our_ambassadors.astro:552]:
```suggestion
```

Comment (christopher-hakkaart) [reply src/pages/our_ambassadors.astro:552]: I'd leave this out until you have alimni. "Soon" or "Coming soon..." feels a little anticapatory.

**Review by christopher-hakkaart — APPROVED** [review]
One minor comment. Not a blocker if you disagree.

File: src/pages/our_ambassadors.astro (hunk 1 above)
Comment (mribeirodantas) [reply src/pages/our_ambassadors.astro:552]: Ok!

**Pull request merged**